src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
```

Comment:
move to html selector

Reply:
**_Hi! Thanks for your advice. 
Why if I apply these rules for tag "body" it doesnt work correctly:_**
body {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

_**But if I do it for "*" everything fine:**_
* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

_**Also where should I put my variable in tag "html" or "body", or i doesn't matter? **_

Reply:
1. Hey, what do you mean "it doesnt work correctly"? Pleas ask this and all other questions in fe_chat 
2. To create a variable with global scope, declare it inside the :root selector. https://www.w3schools.com/css/css3_variables.asp